readme.html

```diff
@@ -1,4 +1,138 @@
+        <ul>
+          <li>Navigate to the <b>Home</b> dropdown and select <b>Apps</b>.</li>
+          <li>Search the Cloud Convert App from the search box.</li>
+          <li>Click on the <b>CONFIGURE NEW ASSET</b> button.</li>
+          <li>Navigate to the <b>Asset Info</b> tab and enter the Asset name and Asset description.
+          </li>
+          <li>Navigate to the <b>Asset Settings</b>.</li>
+          <li>Paste the generated <b>API Key</b> from Cloud Convert UI to its respective configuration parameter.</li>
+          <li>Mention the number of minutes to poll to convert a file. Default is 1 minute.</li>
+          <li>Save the asset.</li>
+          <li>Now, test the connectivity of the Splunk SOAR server to the Cloud Convert instance by clicking the <b>TEST CONNECTIVITY</b> button.</li>
+        </ul>
+      </ul>
+
+    <h2>Explanation of the Asset Configuration Parameters</h2>
+    The asset configuration parameters affect 'test connectivity' and some other actions of the application. The parameters related to test connectivity action are API Key and Number of minutes to poll for the converted file.
+    <ul>
+      <li>
+        <b>API Key: </b> API Token for asset authorization.
+      </li>
+      <li>
+        <b>Number of minutes to poll for converted file: </b> The number of minutes the user wants to take to convert any file. The default time is one minute.
+      </li>
+    </ul>
+  </p>
+  <p>
+    <h2>Explanation of the Cloud Convert Actions' Parameters</h2>
+    <ul>
+      <li>
+        <h3>Test Connectivity (Action Workflow Details)</h3>
+        <ul>
+          <li> This action will test the connectivity of the Splunk SOAR server to the Cloud Convert instance by making an
+            initial API call using the provided asset configuration parameters. </li>
+          <li> The action validates the provided asset configuration parameters. Based on the API call response, the
+            appropriate success and failure message will be displayed when the action gets executed.</li>
+        </ul>
+      </li>
+
+      <li>
+        <h3>Get Valid Filetypes</h3>
+        To retrieve information of valid input-output formats, the user can run this action.
+        <ul>
+          <li><b>Action Parameter</b>: Input filetype</li>
+          This is a required parameter. The input filetype supports 200 filetypes. On selecting any one of the input format, the action generates a custom view listing valid output formats.
+        </ul>
+      </li>
+      <li>
+        <h3>Convert File</h3>
+        This action lets a user convert file for a valid input-output format.
+        <ul>
+          <li><b>Action Parameter: Vault ID</b></li>
+          <ul>
+            <li>This parameter is the unique id for any file and it is a required parameter. In case if file content of more than one file is the same but has a different filename then the vault id will be the same for all those files.</li>
```

Comment:
```suggestion
            <li>This parameter is the unique id for any file and it is a required parameter. In case if file content of more than one file is the same but has a different filename then the vault ID will be the same for all those files.</li>
```

Reply:
Done